applications/MappingApplication/mapping_application.h

```diff
@@ -142,6 +146,11 @@ class KratosMappingApplication : public KratosApplication {
 	///@}
 	///@name Protected member Variables
 	///@{
+  
+  // Needed for Volume Mapping
```

Comment:
I would leave these out and work with Elements (that may be defined in other applications) for Volume mapping.